lib/rubocop/cache_root.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module RuboCop
+  # This class represents the cache root of the caching RuboCop runs.
+  # @api private
+  class CacheRoot
```

Comment:
`CacheRoot` seems a bit too specific to me to warrant a class - maybe this should be `CacheConfig` or something like this?

Reply:
`CacheConfig` looks good! Also changed the class method name from `CacheRoot.dir` to `CacheConfig.root_dir`.